notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -270,3 +270,51 @@ childishness [Andy was rude to the cigar lady and the bartender, and his bellico
 human pleasure [Andy had the pleasure of smoking a good cigar, a rum-tipped Cuban cigar no less.]
 
 
+theamericanshortstory07
+=======================
+
+:: Title
+The Displaced Person
+
+:: Description
+A Holocaust survivor of Polish descent upsets the social order on a Georgia
+farm where he has been hired to work as a farmhand. The story is based on the
+1955 Flannery O'Connor novella of the same name. 
+
+:: Date
+1977-04-12
+
+:: References
+https://www.imdb.com/title/tt0074662/
+
+:: Ratings
+4 <brent>
+3 <paul>
+
+:: Major Themes
+racism in society [The story is set on a Georgia farm in the 1940s and explores the relations between its white owner, Mrs. Mcintyre, and her black workers. Besides the whites repeatedly treating the blacks as an underclass, Mrs. McIntyre berated Mr. Guizac for challenging the social order by arranging for his relative to marry a black man. Mrs. MacIntyre and the Shortleys made bigoted remarks throughout the story.]
+life in Crow law era America [The story is set on a Georgia farm in the 1940s and explores the relations between its white owner, Mrs. Mcintyre, and her black workers.]
+compromising one's ethical principles for personal advantage [Mrs. Mcintyre struggled with whether or not to fire the industrious but social order-challenging Mr. Guizac. People in the community pressured Mrs. McIntyre to get rid of Mr. Guizac because he upset the social fabric, but on the other hand she wanted to keep him on because he was a model farmhand.]
+superior and subordinate [The story follows Mrs. Mcintyre was bossing around the workers on her farm throughout the story.]
+religious hypocrisy [Mrs. McIntyre spoke often of her Christian values in hiring black people and refugees, but crapped on them and criticized them when it suited her. In an ultimate act of religious hypocrisy, she failed to warn Mr. Guizac that an out of control tractor was rolling his way and he was crushed to death under its wheels as a result.]
+hypocritical character [Mrs. McIntyre spoke often of her Christian values in hiring black people and refugees, but crapped on them and criticized them when it suited her.]
+what it is like to be a refugee [The titular "displaced person" Mr. Guizac relocated with his family to a farm in Georgia in the aftermath of the Holocaust.]
+
```

Comment:
I wonder if "treatment of migrants" might also be a theme in the story?

Reply:
I was looking at that one, but decided not to use it on account that we didn't see anything about the government's role in all this.